api/src/main/java/org/pmiops/workbench/utils/mappers/LeonardoMapper.java

```diff
@@ -111,16 +113,31 @@ LeonardoPersistentDiskRequest toLeonardoPersistentDiskRequest(
   @Mapping(target = "creator", source = "auditInfo.creator")
   @Mapping(target = "createdDate", source = "auditInfo.createdDate")
   @Mapping(target = "dateAccessed", source = "auditInfo.dateAccessed")
+  // these 2 values are set by listDisksAfterMapper()
   @Mapping(target = "appType", ignore = true)
   @Mapping(target = "gceRuntime", ignore = true)
   Disk toApiListDisksResponse(LeonardoListPersistentDiskResponse disk);
 
+  @Mapping(target = "creator", source = "auditInfo.creator")
+  @Mapping(target = "createdDate", source = "auditInfo.createdDate")
+  @Mapping(target = "dateAccessed", source = "auditInfo.dateAccessed")
+  // these 2 values are set by listDisksAfterMapper()
+  @Mapping(target = "appType", ignore = true)
+  @Mapping(target = "gceRuntime", ignore = true)
+  Disk toApiListDisksResponse(ListPersistentDiskResponse disk);
```

Comment:
We prepend "Leonardo" to all data structures for the old generated client - see L119.  The new client does not have this, so this is a different class name here.